libs/components/src/navigation/nav-item.component.html

```diff
@@ -0,0 +1,78 @@
+<div
+  class="tw-relative {{
+    active && !hideActiveStyles ? 'tw-bg-background-alt4' : 'tw-bg-background-alt3'
+  }} {{ fvwStyles$ | async }}"
```

Comment:
We can `ngClass` which is a bit cleaner, or we could look into moving this to the typescript code.

```suggestion
  class="tw-relative"
  [ngClass]="[
    active && !hideActiveStyles ? 'tw-bg-background-alt4' : 'tw-bg-background-alt3',
    fvwStyles$ | async
  ]"
```